.github/workflows/docker-images.yaml

```diff
@@ -23,8 +23,14 @@ on:
       - ".github/workflows/docker-images.yaml"
       - "ui/**"
 
-  # On workflow_dispatch, push sha and branch patterns to prefect-dev
+  # On workflow_dispatch, allow publishing 3-latest images
   workflow_dispatch:
+    inputs:
+      publish_3_latest:
+        description: 'Publish 3-latest images'
+        required: true
```

Comment:
why is this required if it defaults to `false`?

Reply:
good point! updated in [bb80382](https://github.com/PrefectHQ/prefect/pull/14633/commits/bb80382718262e3f7d8977cbb5fe111d2ebaf376)